plugins/joblib_launcher/hydra_plugins/joblib_launcher/conf/hydra/launcher/joblib.yaml

```diff
@@ -0,0 +1,16 @@
+hydra:
+  launcher:
+    class: hydra_plugins.joblib_launcher.JoblibLauncher
+    params:
+      joblib:
+        n_jobs: -1
+        backend: None
+        prefer: processes
+        require: None
+        verbose: 0
+        timeout: None
+        pre_dispatch: 2*n_jobs
```

Comment:
is the string `2*n_jobs` being handled by joblib?

Reply:
Yes, Joblib handles the string